src/fheroes2/maps/maps_tiles.h

```diff
@@ -305,7 +305,7 @@ namespace Maps
         TilesAddon * FindFlags( void );
 
         // correct flags, ICN::FLAGS32 vesion
-        void CorrectFlags32( u32 index, bool );
+        void CorrectFlags32( const int col, const u32 index, const bool );
```

Comment:
```suggestion
        void CorrectFlags32( const int col, const u32 index, const bool up );
```

Since we are changing it anyway